plugins/ipam/static/main.go

```diff
@@ -161,7 +161,7 @@ func LoadIPAMConfig(bytes []byte, envArgs string) (*IPAMConfig, string, error) {
 
 				ip, subnet, err := net.ParseCIDR(ipstr)
 				if err != nil {
-					return nil, "", fmt.Errorf("invalid CIDR %s: %s", ipstr, err)
+					return nil, "", fmt.Errorf("the 'ip' field is expected to be in CIDR notation, got: '%s'", ipstr)
```

Comment:
IMO, the new error message just make me more confused :), how about simplify to `failed to parse cidr: %v`

Reply:
+1. That was my original suggestion, but @dougbtv was keen on indicating which attribute was the offending one.

I can reset this back to `fmt.Errorf("Failed to parse CIDR: %s", ipstr)` . Would that work for both of you @dougbtv / @mars1024 ?

Reply:
This is the current implementation I'm proposing.

@mars1024 , your thoughts ? fwiw, I agree with you. There's only CIDR looking thing on the spec, the users shouldn't be confused.

Reply:
Agreed on @dougbtv , let's stay the way now.

Reply:
OK; resolving this item now.